prow_config.yaml

```diff
@@ -33,7 +33,7 @@ workflows:
       - istio/*
       - jupyter/*
       - katib/*
-      - kfdef/*
+      - kfdef/kfctl_istio_dex*
```

Comment:
em. I thought `kfdef/*` is a super set of `kfdef/kfctl_istio_dex*`. Any reason to change it?

Reply:
So if cloud providers are changing `kfctl_aws` / `kfctl_ibm`, `kfctl_openshift`, it should not kick off E2E test because it's not agnostic change.

And I found folks may change very frequently and this change can help speed up and reserve resources.

Reply:
Got it. Make sense.  `kfctl_istio_dex ` should be good enough.